components/brave_rewards/core/wallet_provider/linkage_checker.cc

```diff
@@ -79,8 +79,7 @@ void LinkageChecker::MaybeUpdateExternalWalletStatus(
       client().ExternalWalletDisconnected();
       wallet = nullptr;
     } else {
-      LogError(FROM_HERE) << "Failed to transition " << wallet->type
-                          << " wallet state";
+      LogError(FROM_HERE) << "Failed to transition";
```

Comment:
"Failed to transition wallet state" 👍

Reply:
Done. Thanks Kevin for picking up this change for review so quickly.